app/controllers/document_associations_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class DocumentAssociationsController < ApplicationController
+  rescue_from GdsApi::BaseError do
+    render 'edit_api_down', status: 503
```

Comment:
We should probably log this error, if we don't I'm not sure Sentry will pick it up.

Reply:
Done done :-)